### DISCUSSION TITLE
Outdated doc for AlphaTex regarding the support of multi-track?

### DISCUSSION THREAD

**Discussion by lannka** [CoderLine/alphaTab#572] [Q&A] [answered]
I'm wondering if the statement as highlighted below still holds true. With `data-tracks="all`  attribute, seems multi-track is already supported with AlphaTex?

<img width="846" alt="Screen Shot 2021-04-02 at 10 11 50 AM" src="https://user-images.githubusercontent.com/8496897/113437768-1ca9bf80-939c-11eb-95f8-a21bba0e1444.png">

https://www.alphatab.net/docs/alphatex/introduction

**Comment by Danielku15** [comment, marked as answer]
First of all: welcome to alphaTab. Nice to see a new face here. 😁  

For the future: please directly create a discussion. 😞 I think our contribution information in this repo is quite clear on the fact that when you have general questions, you should create a discussion. Also the templates mention that the template usage is mandatory. It is really exhausting that so many people are simply ignoring all the information and just drop an issue ignoring all my remarks 😭. I migrated the issue to a discussion this time. 

On topic: indeed the documentation is outdated. Multitrack and multi-staff rendering was added as part of https://github.com/CoderLine/alphaTab/issues/207 and it is available since the 1.0 release. There is an own chapter about tracks and staves in the alphaTex docs: https://www.alphatab.net/docs/alphatex/tracks-staves

**Reply by lannka** [reply]
Thank you for the quick answer and apologize for not following the rules here. 
Should the doc be fixed then?

**Reply by Danielku15** [reply]
Just worked on it. Docs are updated as part of https://github.com/CoderLine/alphaTabWebsite/commit/a58c770840bfecd4d64b144291f46c1cec28551c and they will go live in a few mins 😉
